docs/installation.md

```diff
@@ -92,92 +84,51 @@ The tests will perform an initial run and then re-execute each time a project fi
 
 To get started:
 
-- Install [Virtualbox] and [Vagrant] (latest versions recommended).
-- Linux only: An nfsd server is required. You can install this on Ubuntu by running `apt-get install nfs-common nfs-kernel-server`
+- Install Docker & docker-compose (both are installed if using Docker for Windows/Mac).
 
-### Setting up Vagrant
+- If you just wish to [run the tests](common_tasks.md#running-the-tests),
+  you can stop now without performing the remaining steps.
+
+### Starting a local Treeherder instance
```

Comment:
I'm not seeing a note about running the backend with a preset environment variable - `DATABASE_URL`-  and then what command to use to run the frontend.